### PR TITLE
CCITCARBON-367 Fixed notification to display passed and failed tasks for the entire scenario_graph

### PR DESCRIPTION
**Pull request by rujutashinde**

### CODE REVIEW

File: teflo/teflo.py [hunk 1]
```diff
@@ -470,23 +470,27 @@ def cleanup_helper(self, final_passed_tasks: list, final_failed_tasks: list, sta
     def collect_final_passed_failed_tasks_status(self, final_passed_tasks: list,
                                                  final_failed_tasks: list, status: list):
         """
-        This method collects all tests from all scenarios from
+        This method collects all failed ans passed tasks from all scenarios in
```
Comment (JunqiZhang0) [inline teflo/teflo.py:473]: typo

File: teflo/teflo.py [hunk 2]
```diff
@@ -566,7 +569,7 @@ def run_helper(self, sc: Scenario = None, tasklist: list = TASKLIST):
 
                         # reload resource objects
                         sc.reload_resources(data)
-
+                        # WHy is there no self.scenario_graph.reload_resources_from_scenario(sc)
```
Comment (JunqiZhang0) [inline teflo/teflo.py:572]: comment?

Comment (rujutashinde) [reply teflo/teflo.py:572]: removed

File: teflo/teflo.py (hunk 1 above)
Comment (rujutashinde) [reply teflo/teflo.py:473]: done

File: tests/functional/test_teflo.py [hunk 3]
```diff
@@ -95,7 +103,7 @@ def test_results_file_property():
     @staticmethod
     def test_artifacts_folder_created():
         teflo = Teflo(data_folder='/tmp')
-        print(teflo.data_folder)
+        #print(teflo.data_folder)
```
Comment (JunqiZhang0) [inline tests/functional/test_teflo.py:106]: comments

Comment (JunqiZhang0) [inline tests/functional/test_teflo.py:106]: comment

**Review by JunqiZhang0 — APPROVED** [review]
LGTM

**Pull request merged**